File: frontend/src/api/customer-settle/sheet/model/queryCustomerSaleSettleInfoVo.ts
import { SortPageVo } from '@/api/model/sortPageVo';

/** 客户销售结算工作台查询条件。 */
export interface QueryCustomerSaleSettleInfoVo extends SortPageVo {
  /** 单据号。 */
  code?: string;

  /** 客户 ID。 */
  customerId?: string;

  /** 业务类型：1-销售出库单，2-销售退货单。 */
  bizType: number;
}

File: backend/xingyun-settle/src/main/java/com/lframework/xingyun/settle/vo/sheet/customer/QueryCustomerSaleSettleInfoVo.java
package com.lframework.xingyun.settle.vo.sheet.customer;

import com.lframework.starter.web.core.vo.PageVo;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

/**
 * 客户销售业务单据结算工作台查询条件。
 */
@Data
public class QueryCustomerSaleSettleInfoVo extends PageVo {

  private static final long serialVersionUID = 1L;

  @ApiModelProperty("单号")
  private String code;

  @ApiModelProperty("客户ID")
  private String customerId;

  @ApiModelProperty("业务类型：1-销售出库单，2-销售退货单")
  private Integer bizType;
}
